src/systems/ec/implementations/vanilla/packages/ec-component-link/ec-component-link-print.scss

```diff
@@ -0,0 +1,47 @@
+/**
+* Link component
+* @define link
+*/
+
+// Import base
+@import '@ecl/ec-base/ec-base';
+
+@mixin ecl-link-print($link-color: $ecl-color-primary) {
+  .ecl-link {
+    color: $link-color;
+    page-break-inside: avoid;
+    text-decoration: underline;
+  }
+
+  .ecl-link[href^='http']::after {
```

Comment:
why filter? all links are interesting, aren't they?

Reply:
Not really.
Anchor links for instance have no values.
This filter is what I found when searching for "good practicies" for print CSS. It could be improved, but that seems like a widely approved rule.